product_sequence/models/product_product.py

```diff
@@ -32,7 +32,9 @@ def create(self, vals):
     @api.multi
     def write(self, vals):
         for product in self:
-            if product.default_code in [False, '/']:
+            if (product.default_code in [False, '/'] and
+                    ('default_code' not in vals or
```

Comment:
@pedrobaeza 
I rewrote the condition but your example should have "equals to" instead of "not equals to" :

`
>>> def check_default_code(dict):
...     return ('default_code' not in vals or vals['default_code'] == '/')
... 
>>> vals = {}
>>> check_default_code(vals)
True
>>> vals['default_code'] = '/'
>>> check_default_code(vals)
True
>>> vals['default_code'] = '.'
>>> check_default_code(vals)
False
>>> 
>>> def check_revision_default_code(dict):
...     return vals.get('default_code', '/') != '/'
... 
>>> vals = {}
>>> check_revision_default_code(vals)
False
>>> vals['default_code'] = '/'
>>> check_revision_default_code(vals)
False
>>> vals['default_code'] = '.'
>>> check_revision_default_code(vals)
True
>>> 
>>> def check_revision_v2_default_code(dict):
...     return vals.get('default_code', '/') == '/'
... 
>>> vals = {}
>>> check_revision_v2_default_code(vals)
True
>>> vals['default_code'] = '/'
>>> check_revision_v2_default_code(vals)
True
>>> vals['default_code'] = '.'
>>> check_revision_v2_default_code(vals)
False
`